python/lsst/daf/butler/core/config.py

```diff
@@ -220,7 +220,12 @@ def __initFromYaml(self, stream):
 
     def __getitem__(self, name):
         data = self.data
-        for key in name.split("."):
+        # Override the split for the simple case
+        if name in data:
+            keys = (name,)
```

Comment:
Space after comma?

Reply:
flake8 did not complain. I think flake8 doesn't like lists without a space at the end but tuples are fine.